ddl/placement_policy.go

```diff
@@ -144,11 +146,19 @@ func checkPlacementPolicyExistAndCancelNonExistJob(t *meta.Meta, job *model.Job,
 	return nil, err
 }
 
-func onDropPlacementPolicy(t *meta.Meta, job *model.Job) (ver int64, _ error) {
+func onDropPlacementPolicy(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error) {
 	policyInfo, err := checkPlacementPolicyExistAndCancelNonExistJob(t, job, job.SchemaID)
 	if err != nil {
 		return ver, errors.Trace(err)
 	}
+
+	is := d.infoCache.GetLatest()
+	if len(is.DismissPolicyDependencies(policyInfo.Name.L)) == 0 {
```

Comment:
Does `DismissPolicyDependencies` returns all ids of tables who are using this policy info? If it is yes, I think the condition is not enough, because placement policy can also bind with databases.

Reply:
it's meant to return all of it before.